pkgs/development/libraries/libpsl/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
     lzip
     pkgconfig
     python3
-    valgrind
+    (stdenv.lib.optionalString (!stdenv.isDarwin) valgrind)
```

Comment:
Does this work fine with still passing `"--enable-valgrind-tests"`?

Reply:
Yes, `doCheck` is already disabled on darwin.

Reply:
Ah, right.  It looks a bit weird but OK I guess.